packages/configuration-validation/yarn.lock

```diff
@@ -1159,6 +1159,7 @@ decode-uri-component@^0.2.0:
 deep-extend@^0.6.0:
   version "0.6.0"
   resolved "https://registry.yarnpkg.com/deep-extend/-/deep-extend-0.6.0.tgz#c4fa7c95404a17a9c3e8ca7e1537312b736330ac"
+  integrity sha512-LOHxIOaPYdHlJRtCQfDIVZtfw/ufM8+rVj649RIHzcm/vGwQRXFt6OPqIFWsm2XEMrNIEtWR64sY1LEKD2vAOA==
```

Comment:
Is there a reason only this dependency has integrity sha?

Reply:
I am not sure, I will re-update the lock files and see if it changes

Reply:
With updated lock, the integrity has disappeared. Possibly there is a subtle difference between `yarn add` and `yarn install`? Since I had run `yarn add deep-extend -D` at one point, but then removed the dependency from the package.json (replacing with lodash) and ran `yarn install`.

Reply:
What version of yarn are you running?  I believe the integrity check is a newish feature (yarn 1.10)